test/test_rcall.jl

```diff
@@ -5,7 +5,7 @@ using Phylo
 Rinstalled = false
 try
     using RCall
-    include(Pkg.dir("Phylo", "src/rcall.jl"))
+    include(joinpath(dirname(dirname(@__FILE__)), "src/rcall.jl"))
```

Comment:
If you really want me to do it this way, can you fix it to "../src/rcall.jl". Thx. It didn't error out in the tests because I have a `try` fallback in the tests if RCall or R isn't installed which also catches `rcall.jl` not being found.